src/command.coffee

```diff
@@ -102,7 +102,17 @@ exports.run = ->
   process.argv = process.argv[0..1].concat literals
   process.argv[0] = 'coffee'
 
-  opts.output = path.resolve opts.output  if opts.output
+  if opts.output
+    outputBasename = path.basename opts.output
+    if '.' in outputBasename
```

Comment:
I was afraid of testing this, since this part of the codebase has no tests.

All those cases are ones I didn’t consider. Let me cover them . . .